unime/src-tauri/tests/fixtures/states/five_connections_sort_unicode_random_alphabets_name_az.json

```diff
@@ -0,0 +1,62 @@
+{
+  "profile_settings": {
+    "profile": {
+      "name": "Ferris",
+      "picture": "&#129408",
+      "theme": "system",
+      "primary_did": "did:key:z6MknmKLNs3Ps5MFhB4Fp9HqZT4kanNhyxn5xq211xYFHUPJ"
+    },
+    "sorting_preferences": {
+      "connections": {
+        "sort_method": "name_az",
+        "reverse": false
+      },
+      "credentials": {
+        "sort_method": "name_az",
+        "reverse": false
+      }
+    }
+  },
+  "connections": [
+    {
+      "name": "secreter",
+      "url": "https://cbr.nl",
+      "logo_uri": null,
+      "verified": false,
+      "first_interacted": "2023-11-09T13:52:37.991493008+00:00",
+      "last_interacted": "2023-11-09T13:52:45.868117252+00:00"
+    },
+    {
+      "name": "sécréter",
+      "url": "https://thegym_XYZ.nl",
+      "logo_uri": null,
+      "verified": false,
+      "first_interacted": "2023-11-08T13:52:37.991493008+00:00",
+      "last_interacted": "2023-11-08T13:52:45.868117252+00:00"
+    },
+    {
+      "name": "s\u00e9cr\u00e9ter",
+      "url": "https://goverment.com",
+      "logo_uri": null,
+      "verified": false,
+      "first_interacted": "2023-11-09T13:52:11.986023042+00:00",
+      "last_interacted": "2023-11-09T13:52:19.971552835+00:00"
+    },
+    {
+      "id": "א",
```

Comment:
Should we have _actual_ unicode characters in our source code or can we somehow encode them? (I wouldn't know what that symbol is or how to recreate it)

Reply:
we have actual unicode in the tests, as explained in the comments:
```
    // Example of unicode characters, and how they are understood differently by the computer.
    // This example is also used in the json files
    // let no_unicode = "sécréter";
    // let unicode = "sécréter";
    // assert_ne!(no_unicode, unicode);
    // assert_ne!(no_unicode.len(), unicode.len());
    // The above is directly written as unicode characters.
    // In unicode sequence it is "s\u{00e9}cr\u{00e9}ter"
    ```
Does this resolve your question?